torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -2651,6 +2657,98 @@ Error PyTorchModelLoader::loadSqrt(const torch::jit::Node *ptNode) {
   return addValueMapping(outputs[0], glowNode->getResult());
 }
 
+Error PyTorchModelLoader::loadCmpEq(const torch::jit::Node *ptNode) {
```

Comment:
This and all the other `loadCmp*` are all the exact same except for the class used for `createNodeWithBroadcast<>` -- can we just add/use a single `loadCmp` that's templatized? We could just pass the class we're loading in the template into `createNodeWithBroadcast<>`.

Reply:
Hi @jfix71 , Thanks for the suggestion. 
gt/ge uses lt/le nodes internally. To handle this, I have created a separate template for gt ops.